keyboards/40percentclub/nein/keymaps/via/rule.mk

```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
```

Comment:
Should be named `rules.mk`

Reply:
@fauxpark omg i'm special lol I've updated the file name. thanks!